flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -766,7 +770,57 @@ var A320_Neo_UpperECAM;
                                         action: "MAN ON"
                                     },
                                 ]
-                            }
+                            },
+                            {
+                                message: "EXCESS RESIDUAL PR",
+                                level: 2,
+                                flightPhasesInhib: [1, 2, 3, 4, 5, 6, 7, 8, 9],
+                                isActive: () => (
+                                    Simplane.getIsGrounded() &&
+                                    !this.isEngineRunning(1) && !this.isEngineRunning(2) &&
+                                    Math.abs(this.getCachedSimVar("L:A32NX_PRESS_CABIN_DELTA_PRESSURE", "PSI")) > 0.01
+                                ),
+                                actions: [
+                                    {
+                                        style: "action",
+                                        message: "PACK 1",
+                                        action: "OFF"
+                                    },
+                                    {
+                                        style: "action",
+                                        message: "PACK 2",
+                                        action: "OFF",
+                                    },
+                                    {
+                                        style: "action",
+                                        message: "CABIN CREW",
+                                        action: "ALERT",
+                                    },
+                                ]
+                            },
+                            {
+                                message: "LO DIFF PR",
+                                level: 2,
+                                flightPhasesInhib: [2, 3, 4, 5, 7, 8, 9, 10],
+                                page: "PRESS",
+                                isActive: () => (
+                                    this.getCachedSimVar("L:A32NX_PRESS_CABIN_DELTA_PRESSURE", "PSI") < 1.45 &&
+                                    this.getCachedSimVar("VERTICAL SPEED", "feet per minute") < -500 &&
+                                    this.getCachedSimVar("L:A32NX_PRESS_CABIN_ALTITUDE", "feet") > (this.getCachedSimVar("L:A32NX_OVHD_PRESS_LDG_ELEV_KNOB", "feet") + 1500
+                                        || this.getCachedSimVar("L:A32NX_PRESS_AUTO_LANDING_ELEVATION", "feet") + 1500)
```

Comment:
Similar to the other issue, I would suggest moving this to a single boolean LVar (e.g. `A32NX_PRESS_LOW_DIFF_PRESS`) so that the logic lives in Rust, and the FWS can just consume it (here for now or later through Rust from the individual CPCUs).